src/Card/README.md

```diff
@@ -670,3 +671,53 @@ it is meant to be used as a single horizontal row of Cards, not as a grid. See C
   </Card>
 </CardDeck>
 ```
+
+### Card status
+
+Note that in the example below, the content of `Card` is wrapped inside `Card.Body`. The `d-flex` class is added for the `horizontal` orientation to achieve horizontal variant. The `flex-column` class is added for the main `Card` component.
+
+```jsx live
+() => {
+  const [orientation, setOrientation] = useState('vertical');
+  
+  const handleChangeOrientation = (e) => setOrientation(e.target.value);
+  
+  return (
+    <>
+      <Form.Group>
+        <Form.Label>Choose orientation</Form.Label>
+        <Form.RadioSet
+          name="orientation"
+          onChange={handleChangeOrientation}
+          value={orientation}
+        >
+          <Form.Radio value="vertical">Vertical</Form.Radio>
+          <Form.Radio value="horizontal">Horizontal</Form.Radio>
+        </Form.RadioSet>
+      </Form.Group>
+      <Card orientation={orientation} className="flex-column">
```

Comment:
Is `flex-column` a style property we would be able to programmatically add to `Card` when `orientation == "horizontal"` so the consumer doesn't have to remember to add it?

Reply:
I guess `Card.Status` is a unique case for the `orientation` property because we want to have him always at the bottom. Because of that there are such workarounds with `className="flex-column"` and `d-flex` on the `Card.Body`. Actually passing `orientation={orientation}` is only useful for inner components that depend on the `Card.Context`.